Update intro.md (#370)
OutSystems/docs-product: src/extensibility-and-integration/mobile-plugins/firebase/intro.md

            }
        }

> 1. (Optional) If you want to use different configurations for each environment, repeat steps 2 to 6 for each one. Check how to [Override Mobile Extensibility Configurations](https://success.outsystems.com/documentation/11/managing_the_applications_lifecycle/deploy_applications/override_the_default_mobile_extensibility_configurations) in order for the mobile application to point to the correct configuration files in each environment.

Reviewer: ```suggestion
1. (Optional) If you want to use different configurations for each environment, repeat steps 2 to 6 for each environment. See [Override Mobile Extensibility Configurations](https://success.outsystems.com/documentation/11/managing_the_applications_lifecycle/deploy_applications/override_the_default_mobile_extensibility_configurations) details on pointing the mobile application to the correct configuration files in each environment.
```
